src/main/java/frc/robot/LED.java

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot;
+
+import edu.wpi.first.wpilibj.AddressableLED;
+import edu.wpi.first.wpilibj.AddressableLEDBuffer;
+import frc.robot.Constants.LEDPreset;
+
+public class LED {
```

Comment:
didn't drive team wants some extra lighting modes, should we add those before we merge into main?

Reply:
This is working comp code, we can develop new features in different branch and merge after testing.